.github/workflows/ci-java.yml

```diff
@@ -123,7 +123,7 @@ jobs:
           SHORT_SHA="$(git rev-parse --short HEAD)"
           FULL_BRANCH_TAG="${IMAGE_NAME}:${SANITIZED_REF}" # human readable, branch name tag
           FULL_COMMIT_TAG="${IMAGE_NAME}:${SHORT_SHA}"     # a short sha suffixed image tag
-          docker build . --tag "$FULL_BRANCH_TAG"
+          docker build . --tag "$FULL_BRANCH_TAG" --build-arg BASE_ANSIBLE_TAG="a861fa7"
```

Comment:
Assuming we'll need to remove this before merge; just a note so we don't forget.

Reply:
Correct

Reply:
Maybe it shouldn't be resolved until this is removed?

Reply:
What is the "it" you're referencing? The ticket or something else? @brianburton